pkgs/applications/office/paperless/default.nix

```diff
@@ -0,0 +1,170 @@
+{ stdenv
+, lib
+, writeScript
+, fetchFromGitHub
+, makeWrapper
+
+, python36
+, file
+, imagemagickBig # includes ghostscript
+, optipng
+, poppler
+, tesseract
+, unpaper
+}:
+
+## Usage
+
+# ${pkgs.paperless}/bin/paperless wraps manage.py
+
+# ${pkgs.paperless}/share/paperless/setup-env.sh can be sourced from a
+# shell script to setup a Paperless environment
+
+# paperless.withConfig is a convenience function to setup a
+# configured Paperless instance. (See ./withConfig.nix)
+
+# For WSGI with gunicorn, use a shell script like this:
+#   source ${paperless}/share/paperless/setup-env.sh
+#   PYTHONPATH=$paperlessSrc gunicorn paperless.wsgi
+
+with lib;
```

Comment:
I don't think we need this in more than one place. Please get rid of `with` and use `lib` explicily.